apps/examples/bluetooth/bt_unit_test.c

```diff
@@ -0,0 +1,32 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <string.h>
+#include <debug.h>
+
+#include <tinyara/bluetooth/bluetooth.h>
+
+int bt_unit_test_main(int argc, char **argv)
```

Comment:
This example just prints two lines. Is this necessry?

Reply:
This is initial commit. We will implement it after template code is done.